assign_rights/assemble.py

```diff
@@ -52,9 +55,15 @@ def calculate_dates(self, object, request_start_date, request_end_date):
             object, "end_date", request_end_date, object.end_date_period)
         return object_start, object_end
 
-    def create_json(self):
+    def create_json(self, obj, object_start, object_end):
         """docstring for create_json"""
-    pass
+        if obj.__class__.__name__ == "RightsShell":
+            serializer = RightsShellSerializer(obj, context={"start_date": object_start, "end_date": object_end})
```

Comment:
I'm curious about this approach of passing additional context data into the serializer. It seems to me like you could just set those attributes on `obj` directly and then pass those into the serializer? Would keep your serializer a lot cleaner.